### ISSUE TITLE
Serilog UI version 3 is live!

### ISSUE THREAD

**Issue by followynne** [serilog-contrib/serilog-ui#126]
We're happy to announce that the version 3 of Serilog.UI is finally live!

Version 3 brings a **complete restyle of the UI** application, along with some features long requested 🔥. 

As you migrate to v3, you'll be able to use these new features:

- **mobile/desktop mode!**
  [Mobile version shows logs as data cards, offering a separate UX experience with filters and operation buttons
- **light/dark** mode
- **sorting** => logs can be sorted ASC/DESC, using different log properties
- dates can be displayed in UTC or in the user browser timezone

On the Backend side, we can list the following features:

- **Additional Columns** support! (supported sinks: MsSql, MariaDb, PostgreSql)
- additional UI options to configure the application 
- option to customize the Provider name (see #119)
- Basic Authorization can be managed by using a modal, removing the hardcoding of credentials in the backend [see [here](https://github.com/serilog-contrib/serilog-ui/wiki/Configure#authorization) for implementation details]

## Breaking changes

With v3, the minimum supported .NET version is 6.0.

In the WIKI we cover all the details on how to use/implement the new features we offer, please read it carefully!

A migration guide from v2 to v3 can be found [here](https://github.com/serilog-contrib/serilog-ui/wiki/Migrate-v2-=--v3).

### Some BC Details

We took some time to refactor the public API, doing small adjustments to the main interfaces we expose; the goal was to simplify the maintenance and try to reduce the need of API breaking changes in the future, when we'll add new features.

With this in mind, we also reviewed the Data Providers registration:

- all providers support multiple tables/collections registrations
- the DI setup was moved towards Fluent interface
- Authorization interfaces have been changed, and they must be registered in the DI - read [here](https://github.com/serilog-contrib/serilog-ui/wiki/Configure#authorization) to

Another change you could be interested in:

Home path lands on {ui-path}/ instead of {ui-path}/index.html [the old path now redirects towards {ui-path}/]. 
We applied this change to simplify the routing implementation on the client side.

Again, please read the WIKI to get all the details!

## To summarize

Feel free to use this page to post any comment, question, idea you have on version 3 - we'll try to answer as soon as possible 😄 

Let us know if we find any topic that wasn't added to the documentation or that needs additional info. 

Give version 3 a try and let us know what you think about it! 

We hope you'll like it, and we're keen to hear your opinion 💯
